README.md

```diff
@@ -28,6 +28,9 @@ VoTT helps facilitate an end-to-end machine learning pipeline:
 <!-- command: `markdown-toc -i README.md`-->
 
 <!-- toc -->
+- [VoTT (Visual Object Tagging Tool)](#vott-visual-object-tagging-tool)
```

Comment:
Hmm.. I didn't add this, why did it show up here?

Reply:
i had a similar problem... some extension I was using in VSCode understands the auto-generated `toc` comments and when I save the markdown, it re-gens the toc.  I usually end up stripping it out, using another editor.

Not sure if anyone else is seeing similar behavior. annoying enough that maybe we should just leave it in.